storm track dots follow animation function

jordansread opened the pull request:
![image](https://cloud.githubusercontent.com/assets/2349007/19210605/024d4310-8ced-11e6-83fa-73607ef99db2.png)

~~Need to verify the track data is in the same tz as the precip data~~ fixed

Commit:
updating storm and adding opacity switch

Commit:
animated storm dots

jread-usgs commented:
also need to style the storm...

Commit:
not needed

Commit:
want to update precip

Commit:
merge conflict

Merge branch 'master' of github.com:USGS-VIZLAB/hurricane-matthew

# Conflicts:
#	scripts/visualize/visualize.matthew_map.R

Commit:
sites processing

Commit:
fixes #56 "tz" issue, adds nwis

jread-usgs commented:
This one has:

USGS gages on the map
storm location follows animated frames
clicking a gage takes you to NWIS site

Commit:
time added to map, click to NWIS, etc.

jread-usgs commented:
Added time to map, but it: doesn't have any style on it (how the date is formatter or text css), the way times.json comes into the app is a hack (need @jiwalker-usgs help to figure out how that other json file got into target). 

![image](https://cloud.githubusercontent.com/assets/2349007/19214637/2bcad1d8-8d4e-11e6-84b3-063358a9283c.png)

Commit:
watermark and legend

Commit:
merge conflict

Merge branch 'master' of github.com:USGS-VIZLAB/hurricane-matthew

# Conflicts:
#	viz.yaml
